ios/GoldWallet/Info.plist

```diff
@@ -23,7 +23,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>6.0.0</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
if so, it would be better to make some "not-ignoring" env with MARKETING_VERSION that impacts both Android and ios. With this. we would need to change the version on 1 place more. Besides, if you change the version in ios, please change also in android.